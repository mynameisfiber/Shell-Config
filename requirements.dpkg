# the basics
tmux
vim
python3
git
build-essential
fonts-powerline
cmake
stow
exuberant-ctags

# python stuff
libffi-dev
libreadline-dev
libsqlite3-dev
libbz2-dev
zlib1g-dev
libssl-dev
