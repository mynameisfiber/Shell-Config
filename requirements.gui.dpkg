awesome
i3lock
xautolock
libnotify-bin
network-manager-gnome
unagi
x11-xserver-utils
xbacklight
xclip
zeal

blueman

# audio utils
pulseaudio-utils
pulsemixer
pasystray
pavucontrol

# for lolcommits
ffmpeg

# better touchpad
xserver-xorg-input-synaptics

redshift-gtk

# for python
tk-dev
