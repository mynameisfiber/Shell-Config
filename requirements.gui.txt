git+http://github.com/mynameisfiber/typr
