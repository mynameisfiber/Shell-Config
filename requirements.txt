jedi
pep8
flake8
proselint
virtualenv
ipython
pipenv
magic-wormhole
git+http://github.com/mynameisfiber/showsnearme
black
