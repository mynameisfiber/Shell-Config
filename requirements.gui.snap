chromium
wire
signal-desktop
yakyak
slack --classic
spotify
vlc

# for lolcommits
ruby --classic
